src/Deposit.js

```diff
@@ -100,7 +81,9 @@ export class DepositFactory {
    *         as BN instances.
    */
   async availableSatoshiLotSizes() {
-    return await this.systemContract.getAllowedLotSizes()
+    return (await this.systemContract.methods.getAllowedLotSizes().call()).map(
+      toBN
+    )
```

Comment:
😬 Switching back-and-forth is a pain. 

We do have inconsistent usage of big numbers:

 - we use BigNumber [3x in `tbtc-dapp`](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/keep-network/tbtc-dapp%24+BigNumber&patternType=literal) and [a handful](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/keep-network/tbtc%24+BigNumber&patternType=literal) of times in `tbtc`
 - we use BN [90% of the time in `tbtc` tests](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/keep-network/tbtc%24+BN&patternType=literal#10)

Why BigNumber over BN? 🤔

Reply:
`BigNumber` supports decimal math, `BN` only supports integers. `BN` is more performant. Right tool for the right job, but I think for frontend stuff being able to do decimal math is a strong win---in the case of the dApp, it's necessary, since we do signing fee math and display values in TBTC, not tSats or whatever we want to call them :wink:

Reply:
Yup, thought there was a reason 😉 

Do you think it's a small amount of work? Wondering if I can get the hardware wallet stuff done first before we do this. dApp works currently without it.

Also tSats is palindromic, mmmmm I like it...

Reply:
Yeah let's handle this after.